### ISSUE TITLE
Is it possible to have PS1 and PSP games show up without making bubbles? 

### ISSUE THREAD

**Issue by TXFDA** [BlackSheepBoy69/HexFlow-Launcher-Unofficial-Custom#13]
Simple question I guess. I was using this app, and I really like it. It loads quick, and I like the Switch theme. I was having fun making Switch-style tiles for all my games. 

Thing is, I bought a bigger sd card, and put a LOT more games on my Vita. To use this app properly, I'd have to have over 200 bubbles for PS1/PSP games, on top of the 90-ish Vita games, and another 14 homebrew ports (plus homebrew app stuff, and stock apps). And that's not counting anything else I end up adding later. So, I ended up having to swap back to Retroflow for now, since it doesn't require bubbles to still list the games. I know I could technically fit them all into bubbles, and be under the 500 bubble limit, but still, I don't particularly want over 300 bubbles.

I would love to go back to this, and use the Switch theme again.

Also, I hope this was okay to ask here. I'm not super familiar with github outside of downloading stuff from it. And I didn't see anywhere else to specifically ask questions or make requests for this version of the app. Sorry if I did something wrong. I saw someone else making requests and stuff and thought it'd be okay.

**Comment by BlackSheepBoy69** [comment]
Yes. Support for Adrenaline Launcher and PSP categories Lite is working in the experimental non-public version. It's expected to be public in the next version. It has a lot of dependencies (ex: it messes with your cache a lot and turning it ON/OFF takes a long time) and I'm trying to make it work more cleanly, the same way you can turn RetroFlow ON/OFF and it will load all of them in super clean and fast.

**Comment by TXFDA** [comment]
Cool beans, I look forward to 3.0 then. I appreciate the reply. I'll definitely look at that link too. It sounds useful. 

And definitely wouldn't mind the mod, at least for the mean time, if you don't mind that is.

**Comment by BlackSheepBoy69** [comment]
Seemed to be easy enough to just copy over the few lines of code, but getting a strange error. Probably something really easy to fix. I'll check it out Friday evening or on the weekend if I can!

**Comment by BlackSheepBoy69** [comment]
https://www.mediafire.com/file/u3u3rq4kq0gjq16/TXFDA%2527s_Modpack.rar/file

Instructions are included, the index.lua file inside is the source code. It's pretty easy to install. I was too lazy to make a VPK for you sorry. Feel free to do literally whatever you want with this code as long as you have RetroFlow team's permission, post it, make a VPK, I don't really care. This will be able to tide you over until I finish bugtesting for my final update with the features that are making you want to use RetroFlow instead!

It is a modpack for RetroFlow 6.0.0/6.1.0/6.1.1
Mod list:
- Touch-scrolling smoothness upgrade
- It will always load the SwitchView assets upon startup
- View 5 is replaced with SwitchView
- The SwitchView's bottom menu is fully functional. It can be accessed with touch, analog, or D-pad.
- "DrawCover" function math optimization (there are other things to reduce lag but is the easiest to add)
- THE SHADOW AT THE BOTTOM OF SWITCHVIEW CAN'T BE DISABLED LIKE IT CAN IN HEXFLOW CUSTOM... because I was too lazy to mess with the complicated RetroFlow settings menu. As such I'd recommend using one of the custom backgrounds he provides, such as 'Blur 1'.
- Horizontal visual snap has been reduced so it looks super smooth in SwitchView, but may look worse in the 3D views when RetroFlow's 'smooth scroll' is enabled, and note: it is enabled by default.

**Comment by BlackSheepBoy69** [comment]
https://www.mediafire.com/file/uezpocdeb8rfu2f/index.lua/file

Put this file in "ux0:app/HXLC00001/index.lua" and it will add Adrenaline Launcher to HexFlow Custom v2.2

Putting "Adrenaline Launcher: ON" in settings will slow loading time by about 1 second. This build is fairly stable but lacks support for PSP Categories Lite, lacks the ability to rename/override the PSP/PS1 games, lacks the overhaul for custom categories, and lacks the ability to have the extra RetroFlow categories like MAME. Do not override Vita or homebrews to the PSP/PS1 category as they will effectively be hidden in this experimental build.

It will be cool if you try it out and let me know any feedback, because I feel it should be pretty stable! Thank you in advance if you do.

**Comment by BlackSheepBoy69** [comment]
Support for Adrenaline Launcher (either from ux0: or uma0: ) has been added in v2.8 and above. PSP and PS1 can now launch without bubbles.
